src/libraries/System.Security.Cryptography.Algorithms/tests/Rfc2898Tests.cs

```diff
@@ -503,6 +484,16 @@ private static IEnumerable<KnownValuesTestCase> GetKnownValuesTestCases()
                 AnswerHex = "3D2EEC4FE41C849B80C8D83662C0E44A8B291A964CF2F07038",
             };
 
+            yield return new KnownValuesTestCase
+            {
+                CaseName = "RFC 6070 Case 1",
```

Comment:
My obsessive nature wants this above 6070#5 (the previous case), but if this is the only reason to spin the build I wouldn't bother.

Reply:
I looked through my PR again and realized I never removed to localized exception, so, let's use that as a reason to push and fix the nits while we are at it.